### DISCUSSION TITLE
Just Created a Directory of 86 Health Plan MRFs - What Plans Are Missing??

### DISCUSSION THREAD

**Discussion by numbercruncher01** [CMSgov/price-transparency-guide#678] [General]
👀 Looking for a one-stop-shop 🎯 for all Health Plan MRF URLs/Links? Your search ends here! 👉 https://brightspotinsights.com/903ac837f6334698969b3e238191a633?v=6d30b328ba2446d7b32ff0f79f3cbfdc

💻 Feel free to share with anyone who would find this helpful.

I'm passionately 🛠️ building out this website as a valuable resource for everyone navigating the world of Payer Machine Readable Files.

Starting off, I've collated the URLs for 86 health plans. Although it doesn't encompass every health plan just yet, I'm committed to reaching that milestone in the coming weeks. 📈 What Health Plans am I missing??

Thanks,
Brian

**Comment by tonicwave** [comment]
Thanks, Brian!

This is definitely a useful contribution to the world.

**Reply by numbercruncher01** [reply]
Thanks...I appreciate the support!
